Dockerfile

```diff
@@ -67,6 +72,7 @@ RUN apt-get update && apt-get install -y \
 	xfsprogs \
 	libzfs-dev \
 	tar \
+	linux-image-`uname -r` \
```

Comment:
I'm running Arch Linux with a custom kernel and the ``` linux-image-`uname -r` ```  dependency breaks the build for me since there isn't a Ubuntu a package that matches my kernel.  Without it the build is fine for me.  I'm not sure what situation this is needed for, but it might be good to use a different technique that still works for users who aren't running a Ubuntu kernel on their host system.